builder/dockerfile/internals.go

```diff
@@ -137,6 +171,53 @@ func (b *Builder) performCopy(state *dispatchState, inst copyInstruction) error
 	return b.exportImage(state, imageMount, runConfigWithCommentCmd)
 }
 
+func parseChownFlag(flChown string) (string, string) {
```

Comment:
because the context is quite different and requires different *knowledge* in each situation where it is used. `pkg/idtools` only considers the case where the host `/etc/{passwd,group}` files are used for translation, as well as possible use of `getent`. That would be totally invalid for this case, where we need the **container** files to be used.